store/persistent/benchmarks.go

```diff
@@ -105,6 +107,76 @@ func BenchParallelGet(s store.StorageFlusher, size string, b *testing.B) {
 	wg.Wait()
 }
 
+func BenchCidGet(s store.PersistentStorage, b *testing.B) {
+	cids, err := utils.RandomCids(1)
+	if err != nil {
+		panic(err)
+	}
+	p, _ := peer.Decode("12D3KooWKRyzVWW6ChFjQjK4miCty85Niy48tpPV95XdKu1BcvMA")
+
+	entry := store.MakeIndexEntry(p, protocolID, cids[0].Bytes())
+
+	cids, _ = utils.RandomCids(4096)
+	s.PutMany(cids, entry)
+
+	for testCount := 1024; testCount < 10240; testCount *= 2 {
+		b.Run(fmt.Sprint("Get", testCount), func(b *testing.B) {
+			b.ReportAllocs()
+			b.ResetTimer()
+			m := initMetrics()
+			for i := 0; i < b.N; i++ {
+				for j := 0; j < testCount; j++ {
```

Comment:
These two loops are the equivalent of:
```go
for i := 0; i < b.N * testCount; i++ {
    ...
}
```
Makes more sense to let `b.N` alone vary the number of queries.  That way the benchmark can vary `b.N` to see how long it takes to do a query on average.